test/a11y-wand-test.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Tests for a11yTextWand
+ */
+
+let assert = require("assert");
+let mockDom = require("./mock-dom");
+
+describe("Screen Reader Wand", function() {
+    let plugin = null;
+
+    before(function(done) {
+        mockDom.createDom(function() {
+            let ScreenReaderWand = require("../plugins/a11y-text-wand");
+            plugin = new ScreenReaderWand();
+
+            document.body.innerHTML = `
+                <article id="article">
+                    <p>This paragraph is visible.</p>
+                    <p id="p-with-hidden">This paragraph has a <span id="span" aria-hidden="true">hidden</span> span.</p>
+                    <p id="hidden-p" aria-hidden="true">This paragraph is hidden.</p>
+                </article>
+            `;
+
+            done();
+        });
+    });
+
+    it("should handle aria-hidden correctly", function() {
+        plugin.run();
+
+        $(document).trigger("mousemove.wand", $("#span"));
+
+        assert($(".tota11y-info-section").text() === "No text visible to a screen reader");
```

Comment:
NIT: Consider using the AAA commentary:

```
// Arrange

// Act

// Assert
```

Reply:
I can add the comments if you prefer. I was trying to follow the same style as the rest of the tests in the repo as much as possible.

This one does sort of break that trend by generating the markup in the `before()`, but since the same markup applies to each test, it seemed better to create it once and reuse it.

Reply:
I hope to change the rest of the tests too, soon :smile:

Thank you for leading the charge!

Reply:
I went ahead and amended the previous commit with the Arrange, Act, Assert comments for the new tests.